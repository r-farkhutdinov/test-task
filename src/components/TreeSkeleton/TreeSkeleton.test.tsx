import { render } from '@testing-library/react';
import { TreeSkeleton } from '.';

describe('TreeSkeleton component', () => {
    it('matches snapshot without count prop', () => {
        const { asFragment } = render(<TreeSkeleton />);
        expect(asFragment()).toMatchSnapshot();
    });

    it('matches snapshot with count prop', () => {
        const { asFragment } = render(<TreeSkeleton count={5} />);
        expect(asFragment()).toMatchSnapshot();
    });
});
