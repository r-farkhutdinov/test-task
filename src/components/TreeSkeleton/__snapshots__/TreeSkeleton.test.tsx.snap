// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TreeSkeleton component matches snapshot with count prop 1`] = `
<DocumentFragment>
  <div
    class="root"
    data-testid="skeleton"
    style="margin-left: 0px; margin-right: 0px;"
  />
  <div
    class="root"
    data-testid="skeleton"
    style="margin-left: 16px; margin-right: 32px;"
  />
  <div
    class="root"
    data-testid="skeleton"
    style="margin-left: 32px; margin-right: 0px;"
  />
  <div
    class="root"
    data-testid="skeleton"
    style="margin-left: 0px; margin-right: 32px;"
  />
  <div
    class="root"
    data-testid="skeleton"
    style="margin-left: 16px; margin-right: 0px;"
  />
</DocumentFragment>
`;

exports[`TreeSkeleton component matches snapshot without count prop 1`] = `
<DocumentFragment>
  <div
    class="root"
    data-testid="skeleton"
    style="margin-left: 0px; margin-right: 0px;"
  />
  <div
    class="root"
    data-testid="skeleton"
    style="margin-left: 16px; margin-right: 32px;"
  />
  <div
    class="root"
    data-testid="skeleton"
    style="margin-left: 16px; margin-right: 0px;"
  />
  <div
    class="root"
    data-testid="skeleton"
    style="margin-left: 16px; margin-right: 32px;"
  />
  <div
    class="root"
    data-testid="skeleton"
    style="margin-left: 32px; margin-right: 0px;"
  />
  <div
    class="root"
    data-testid="skeleton"
    style="margin-left: 32px; margin-right: 32px;"
  />
  <div
    class="root"
    data-testid="skeleton"
    style="margin-left: 32px; margin-right: 0px;"
  />
  <div
    class="root"
    data-testid="skeleton"
    style="margin-left: 32px; margin-right: 32px;"
  />
  <div
    class="root"
    data-testid="skeleton"
    style="margin-left: 0px; margin-right: 0px;"
  />
  <div
    class="root"
    data-testid="skeleton"
    style="margin-left: 0px; margin-right: 0px;"
  />
</DocumentFragment>
`;
